helpdesk/management/commands/get_email.py

```diff
@@ -400,8 +400,11 @@ def ticket_from_message(message, queue, logger):
         mail = BeautifulSoup(part.get_payload(), "lxml")
         if ">" in mail.text:
             body = mail.find('body')
-            body = body.text
-            body = body.encode('ascii', errors='ignore')
+            try:
+               body = body.text
+               body = body.encode('ascii', errors='ignore')
+            except AttributeError:
+               body = ''
```

Comment:
I think it would be better to default to `body = mail.text`.

Reply:
In a first time i think like you and make `body = mail.text` but i got html code into the ticket.
so i set body to `''`.

Reply:
It seems to me like that's better than showing nothing at all.

Reply:
We could maybe use BeautifulSoup such as `mail.get_text()` to try to filter out any html present.